upgrade-tidb-using-tiup.md

```diff
@@ -6,20 +6,26 @@ aliases: ['/docs/dev/upgrade-tidb-using-tiup/','/docs/dev/how-to/upgrade/using-t
 
 # Upgrade TiDB Using TiUP
 
-This document is targeted for users who want to upgrade from TiDB 4.0 versions to TiDB 5.0 versions, or from TiDB 5.0 to a later version.
+This document is targeted for the following upgrade paths:
 
-If your cluster to upgrade is v3.1 or an earlier version (v3.0 or v2.1), you need to upgrade your cluster first to v4.0 and then to v5.0. You cannot upgrade your cluster across a major version or versions.
+- Upgrade from TiDB 4.0 versions to TiDB 5.1 versions, or from TiDB 5.1 to a later v5.1 patch version.
+- Upgrade from TiDB 5.0 versions to TiDB 5.1 versions, or from TiDB 5.1 to a later v5.1 patch version.
```

Comment:
```suggestion
- Upgrade from TiDB 4.0 versions to TiDB 5.1 versions.
- Upgrade from TiDB 5.0 versions to TiDB 5.1 versions,
```

Reply:
perhaps, we could mention " Upgrade TiDB 5.1 to a later v5.1 patch version. " as the 3rd item after we have the first 5.1 patch version?